website/docs/docs/deploy/deploy-environments.md

```diff
@@ -41,6 +41,46 @@ In dbt Cloud, each project can have one designated deployment environment, which
 
 For Semantic Layer-eligible customers, the next section of environment settings is the Semantic Layer configurations. [The Semantic Layer setup guide](/docs/use-dbt-semantic-layer/setup-sl) has the most up-to-date setup instructions!
 
+## Staging environment <Lifecycle status='beta' />
+
+:::note
+Currently in limited availability beta. Contact support or your account team if you're interested in beta access.
+:::
+
+Use a Staging environment to grant developers access to deployment workflows and tools while controlling access to production data. Configure Staging with a long-living branch (for example, `staging`) similar to but separate from the primary branch (for example, `main`).
```

Comment:
From initial customer beta feedback, I think there are two legitimate patterns:
1. Production = default branch (`main`)
    - point Development + Staging to `staging` branch
    - point Production to `main`
    - promote to Production by merging from `staging -> main`
2. Staging = default branch (`main`)
    - point Development + Staging to `main` branch
    - promote to Production by cutting "release" branches/tags off `main` at a regular interval (daily, weekly)
    - update Production environment (via [dbt Cloud Admin API](https://docs.getdbt.com/dbt-cloud/api-v3#/operations/Update%20Environment)) to point to latest "release" branch/tag

Right now we've documented (1) here. I don't want to add too much complexity here, but we should mention that it's not the only legitimate/supported pattern.

Reply:
@jtcohen6 I added that there are other ways. I feel like both of these should be captured as procedures, but I'm not really sure whether they fit in that page or require their own. I will create an issue for this.